app-react/src/App.test.tsx

```diff
@@ -1,9 +1,43 @@
-import React from 'react';
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import { TransactionView } from './components/TransactionView';
+import { TransactionType } from '../../app-express/src/api/types';
+import { Transaction } from '../../app-express/src/api/types';
 
```

Comment:
I have taken my testing philosophy from Kent C Dodds:

https://kentcdodds.com/blog/static-vs-unit-vs-integration-vs-e2e-tests
https://kentcdodds.com/courses

So just testing input/output in terms of props => html in the case of testing components.